src/issue/screens/issue.screen.js

```diff
@@ -314,14 +341,20 @@ class Issue extends Component {
     );
   };
 
+  renderFooter = () => {
+    return this.props.timelineItemsPagination.nextPageUrl ? (
+      <LoadingCommonItem />
```

Comment:
This is for showing the loading indicator at the bottom when user is pulling the screen up to scroll more right? Maybe we should rename this to `LoadingFooterIcon` or something similar?

Tiny nit though

Reply:
Naming is one of the most difficult things in computer science. 🙈 
I saw other loading components were called `loading-xx-item`, and thought this one was more in common use. BOOM, `loading-common-item`.